GameData/RealismOverhaul/RO_SuggestedMods/HabTech2/RO_HabTech2.cfg

```diff
@@ -1,157 +1,807 @@
-//	Leonardo Permanent Multipurpose Module (PMM)
-//	Diameter: 4.57m
-//	Length: 6.6m
-//	Mass: 4.437944128t
-//	Source: https://en.wikipedia.org/wiki/Leonardo_(ISS_module)
 
-@PART[ht2_MPLM]:FOR[RealismOverhaul]
```

Comment:
Does this part `ht2_MPLM` no longer exist?

Reply:
yes it does, I think its just removed it from that line as in my configs its on a different line.

Reply:
Ok, I see it.  For my sanity in reviewing, can you move it back up to the top?